blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,224 @@
+---
+title: The workspaces of Sourcegraph
+author: Rob Rhyne
+authorUrl: https://twitter.com/robrhyne
+publishDate: 2021-02-10T18:00+02:00
+tags: [blog, remote, design]
+slug: workspaces-of-sourcegraph
+published: true
+---
+
+The Sourcegraph design team is exploring how to improve the display of code throughout the application. As part of this exploration, we are reviewing trends in typography, color schemes, and UI complexity in the places our engineers view code the most… their editors.  While our research is far from complete, we thought you might find a breakdown of the visual characteristics of where our team writes code of interest. 
+
+## Light/dark mode and contrast
+Three quarters of our engineers preferred a dark theme over a light theme.  Curiously, the majority of dark mode users chose a low contrast between code and background. Backgrounds colors averaged 85% black while the  primary code color to background contrast ratio was around 9 to 1. 
```

Comment:
This is a great data point! Can you lead into it? What is interesting about studying light/dark mode and contrast? Why is it "curious" that dark mode users chose a low contrast?  The answer may seem intuitive or obvious to you but including a sentence explaining will make sure the % of readers who don't intuitively know this is interesting stay with you. Keeping it to about a sentence will make sure readers who do share your intuition are not bored.